src/client.js

```diff
@@ -1310,7 +1310,6 @@ wrapCryptoFuncs(MatrixClient, [
     "bootstrapSecretStorage",
     "addSecretStorageKey",
     "hasSecretStorageKey",
-    "secretStorageKeyNeedsUpgrade",
```

Comment:
Maybe should keep this in order to not break backwards compat, but just have it return `false`.

Reply:
I'd rather remove it and mark this as a breaking change tbh. Will do so in the PR description :+1:

Reply:
About breaking changes, we've been marking 4S / cross-signing APIs in `client.js` with the text "The Secure Secret Storage API is currently UNSTABLE and may change without notice.", so I think that may be enough wiggle room to make breaking changes like this without a major version bump.

Reply:
Filed https://github.com/vector-im/riot-web/issues/13651 to track properly stabilising these APIs.